peoplepage.html

```diff
@@ -77,138 +103,141 @@
-            row.classList.add('row');
-
-            artists.forEach((artist, index) => {
-                const artistCard = document.createElement('div');
-                artistCard.classList.add('col-md-2', 'artist-card');
-
-                const profileImage = document.createElement('img');
-                profileImage.src = `https://image.tmdb.org/t/p/w200${artist.profile_path}`;
-                profileImage.alt = artist.name;
-
-                const artistName = document.createElement('p');
-                artistName.textContent = artist.name;
-                artistName.classList.add('artist-name');
-
-                artistCard.appendChild(profileImage);
-                artistCard.appendChild(artistName);
-
-                artistCard.addEventListener('click', () => {
-                    const artistId = artist.id;
-                    const artistInfoUrl = `https://api.themoviedb.org/3/person/${artistId}?api_key=${apiKey}`;
-                    window.location.href = `artist.html?id=${artistId}&url=${artistInfoUrl}`;
-                });
 
-                row.appendChild(artistCard);
+        function displayArtists(artists) {
+            artistsContainer.innerHTML = '';
+
+            if (artists.length === 0) {
+                const artistNotFoundMessage = document.createElement('p');
+                artistNotFoundMessage.textContent = 'Artist not found.';
+                artistsContainer.appendChild(artistNotFoundMessage);
+            } else {
+                let row = document.createElement('div');
+                row.classList.add('row');
+
+                artists.forEach((artist, index) => {
+                    const artistCard = document.createElement('div');
+                    artistCard.classList.add('col-md-2', 'artist-card');
+
+                    const profileImage = document.createElement('img');
+                    profileImage.src = `https://image.tmdb.org/t/p/w200${artist.profile_path}`;
+                    profileImage.alt = artist.name;
+
+                    const artistName = document.createElement('p');
+                    artistName.textContent = artist.name;
+                    artistName.classList.add('artist-name');
+
+                    artistCard.appendChild(profileImage);
+                    artistCard.appendChild(artistName);
+
+                    artistCard.addEventListener('click', () => {
+                        const artistId = artist.id;
+                        const artistInfoUrl = `https://api.themoviedb.org/3/person/${artistId}?api_key=${apiKey}`;
+                        window.location.href = `artist.html?id=${artistId}&url=${artistInfoUrl}`;
+                    });
+
+                    row.appendChild(artistCard);
+
+                    if ((index + 1) % 5 === 0) {
+                        artistsContainer.appendChild(row);
+                        row = document.createElement('div');
+                        row.classList.add('row');
```

Comment:
same as above, adding the class to center the artist elements

Reply:
done